### DISCUSSION TITLE
Taro会支持明年的鸿蒙next吗，会以小程序的方式还是RN的方式支持呢

### DISCUSSION THREAD

**Discussion by cocolbell** [NervJS/taro#15198] [Feature Request]
### 这个特性解决了什么问题？
鸿蒙next平台的开发

### 这个 API 长什么样？
看今年的鸿蒙开发者大会明确taro在鸿蒙next的支持范围内，想确认下是否会支持，如果支持会以什么样的方式支持（鸿蒙next据说会支持ReactNative）

同时了解下能否参与共建

**Comment by ZakaryCode** [comment]
有个文档的 PR 或许能帮到你 https://github.com/NervJS/taro-docs/pull/199

**Comment by cocolbell** [comment]
> 有个文档的 PR 或许能帮到你 [NervJS/taro-docs#199](https://github.com/NervJS/taro-docs/pull/199)

666大佬，看文档是直接支持编译成原生ArkUI + ArkTs了，不过里面的版本号都未发布，应该是要等taro 3.8 发布了吧？

**Comment by ZakaryCode** [comment]
> 应该是要等taro 3.8 发布了吧

年底应该会有个 canary 版本

**Comment by SuJunming** [comment]
能先体验Taro尝鲜吗？想申请个资格。

**Comment by ZakaryCode** [comment]
> 能先体验Taro尝鲜吗？想申请个资格。

4.x beta 就可以体验了，需要鸿蒙自己的内测资格
